src/components/Headers/Header.tsx

```diff
@@ -0,0 +1,16 @@
+export default function Header() {
+  return (
+    <div className="flex shadow-sm p-4 justify-end">
+      <div className="flex flex-col items-center space-x-4 text-gray-400 mr-3">
+        <span className="flex flex-col ml-2 h-full">
+          <span className="truncate w-30 text-black font-semibold tracking-wide leading-none">
+            Bupe Mulenga
+          </span>
+          <span className="truncate w-30 text-black text-xs leading-none mt-1">
+            Software Engineer
+          </span>
```

Comment:
lets not hardcode  data  values
we   are at a point where we want to start seeing how actual  data  will be handled.

lets fetch  values from  the API (create some  dummy  data in the admin at  `localhost:8000/admin`)

This applies to all  other places with dummy data

Reply:
@sonlinux I have to fetch real data from db before I can update this PR right?

Reply:
yes  that too

Reply:
so for that user it  must be showing the currently logged in user

Reply:
but firstly merge BUpe's PR with yours locally to see what changed